docs/get_started/quickstart.rst

```diff
@@ -8,32 +8,359 @@ Quick Start
 =====================
 
 |productName| has a variety of APIs to choose from when setting up and running a federation. 
-In this quick start guide, we will demonstrate how to run a simple federated learning example using the Task Runner API and Hello Federation script
+In this quick start guide, we will demonstrate how to run a simple federated learning example using the Task Runner API.
 
-.. note::
 
-    The example used in this section is designed primarily to demonstrate functionality of the package and its components. It is not the recommended method for running a real world federation.
 
-    See :ref:`openfl_examples` for details.
+.. _creating_a_federation:
 
-.. _hello_federation:
+********************************
+Creating a federation in 5 steps
+********************************
 
-*********************************
-Hello Federation
-*********************************
-.. note::
+To begin we recommend installing OpenFL inside a python virtual environment. This can be done with the following:
 
-    Ensure you have installed the |productName| package.
+.. code-block:: console:
+    
+    pip install virtualenv
+    virtualenv ~/openfl-quickstart
+    source ~/openfl-quickstart
+    pip install openfl
 
-    See :ref:`install_package` for details.
 
-We will use the `"Hello Federation" python script <https://github.com/intel/openfl/blob/develop/tests/github/test_hello_federation.py>`_ to quickly create a federation (an aggregator node and two collaborator nodes) to test the project pipeline.
+Now you're ready to run your first federation! Copying these commands to your terminal will run a simple federation with two collaborators all on your local machine. These commands can be broken down into five steps, which you can read more about `here <../about/features_index/taskrunner.html#step-1-create-a-workspace>`_
 
-.. literalinclude:: ../../tests/github/test_hello_federation.py
-  :language: python
+1. Setup Federation Workspace & Certificate Authority (CA) for Secure Communication
+2. Setup Aggregator & Initialize Federation Plan + Model
+3. Setup Collaborator 1
+4. Setup Collaborator 2
+5. Run the Federation
 
-Run the script
+.. code-block:: console
+
+   ############################################################################################

```

Comment:
Suggestion: What do you two think on making these headings show up also in the terminal using echo?
``` 
echo "############################################################################################"
echo "# Step 1: Setup Federation Workspace & Certificate Authority (CA) for Secure Communication #"
echo "############################################################################################"

# Generate an OpenFL Workspace. This example will train a pytorch
# CNN model on the MNIST dataset
fx workspace create --template torch_cnn_mnist --prefix my_workspace
cd my_workspace

# This will create a certificate authority (CA), so the participants communicate over a secure TLS Channel
fx workspace certify

echo "#################################################################"
echo "# Step 2: Setup Aggregator & Initialize Federation Plan + Model #"
echo "#################################################################"

# Generate a Certificate Signing Request (CSR) for the Aggregator
fx aggregator generate-cert-request

# The CA signs the aggregator's request, which is now available in the workspace
fx aggregator certify --silent

# Initialize FL Plan and Model Weights for the Federation
fx plan initialize

echo "################################"
echo "# Step 3: Setup Collaborator 1 #"
echo "################################"

# Create a collaborator named "collaborator1" that will use data path "1"
fx collaborator create -n collaborator1 -d 1

# Generate a CSR for collaborator1
fx collaborator generate-cert-request -n collaborator1

# The CA signs collaborator1's certificate
fx collaborator certify -n collaborator1 --silent

echo "################################"
echo "# Step 4: Setup Collaborator 2 #"
echo "################################"

# Create a collaborator named "collaborator2" that will use data path "2"
fx collaborator create -n collaborator2 -d 2

# Generate a CSR for collaborator2
fx collaborator generate-cert-request -n collaborator2

# The CA signs collaborator2's certificate
fx collaborator certify -n collaborator2 --silent

echo "##############################"
echo "# Step 5. Run the Federation #"
echo "##############################"

# Run the Aggregator
fx aggregator start &

# Run Collaborator 1
fx collaborator start -n collaborator1 &

# Run Collaborator 2
fx collaborator start -n collaborator2

echo "Congratulations! You've run your first federation with OpenFL"
```